packages/gatsby-adapter-netlify/src/image-cdn-url-generator.ts

```diff
@@ -0,0 +1,60 @@
+import type { ImageCdnTransformArgs } from "gatsby"
+
+export function generateImageUrl(
+  source: {
+    url: string
+    filename: string
+    mimeType: string
+    internal: { contentDigest: string }
+  },
+  imageArgs: ImageCdnTransformArgs
+): string {
+  const placeholderOrigin = `http://netlify.com`
+  const imageParams = generateImageArgs(imageArgs)
+
+  const baseURL = new URL(`${placeholderOrigin}/.netlify/images`)
+
+  baseURL.search = imageParams.toString()
+  baseURL.searchParams.append(`url`, source.url)
+  baseURL.searchParams.append(`cd`, source.internal.contentDigest)
+
+  return `${baseURL.pathname}${baseURL.search}`
+}
+
+export function generateImageArgs({
+  width,
+  height,
+  format,
+  cropFocus,
+  quality,
+}: ImageCdnTransformArgs): URLSearchParams {
+  const params = new URLSearchParams()
+
+  if (width) {
+    params.append(`w`, width.toString())
+  }
+  if (height) {
+    params.append(`h`, height.toString())
+  }
+  if (cropFocus) {
+    params.append(`fit`, `crop`)
+    if (Array.isArray(cropFocus)) {
+      // For array of cropFocus values, append them as comma-separated string
+      params.append(`crop`, cropFocus.join(`,`))
+    } else {
+      params.append(`crop`, cropFocus)
```

Comment:
I wonder if we want to add any kind of comment here that we are using a field that is supported but not documented with `crop` (`position` is documented but both work with the api).

Reply:
is it just alias and we could just use `position` instead of `crop` to use documented one?